### PR TITLE
Fixes Dropdown wrong default color

### LINKED ISSUE #39: The default color of the CascadeDropdownMenu is not the same as DropdownMenu
Before:
image

after:
image

Showing a default DropdownMenu vs CascadeDropdownMenu

https://github.com/saket/cascade/assets/67873169/4008fd78-c6d5-42e7-9202-a39f8f033dcd

I am also not sure how not to change the default color with CascadeDropdownMenu

<details> <summary> code in question </summary>

```kotlin

    CascadeDropdownMenu(
        expanded = expanded,
        onDismissRequest = onDismissRequest,
        modifier = Modifier.semantics { testTagsAsResourceId = true },
    ) {

        DropdownMenuItem(
            text = { Text(text = stringResource(R.string.home_refresh)) },
            leadingIcon = { Icon(Icons.Outlined.Refresh, contentDescription = null) },
            onClick = {
                onDismissRequest()
                onClickRefresh()
            },
            modifier = Modifier.testTag("jerboa:refresh"),
        )
        DropdownMenuItem(
            text = { Text(text = stringResource(R.string.home_post_view_mode)) },
            leadingIcon = { Icon(Icons.Outlined.ViewAgenda, contentDescription = null) },
            children = {
                PostViewMode.entries.map {
                    DropdownMenuItem(
                        text = { Text(text = stringResource(it.mode)) },
                        onClick = {
                            onClickPostViewMode(it)
                            onDismissRequest()
                                  },

                        modifier =
                        if (selectedPostViewMode == it) {
                            Modifier.background(MaterialTheme.colorScheme.onBackground.copy(alpha = .1f))
                        } else {
                            Modifier
                        }.testTag("jerboa:postviewmode_${it.name}"),
                    )
                }
            },
            modifier = Modifier.testTag("jerboa:postviewmode"),
        )
        DropdownMenuItem(
            text = { Text(stringResource(R.string.home_site_info)) },
            leadingIcon = { Icon(Icons.Outlined.Info, contentDescription = null) },
            onClick = {
                onClickSiteInfo()
                onDismissRequest()
            },
        )
```        

Old version https://github.com/MV-GH/jerboa/blob/a5d1b01a6b219ad524d5d97e897e6514058646d5/app/src/main/java/com/jerboa/ui/components/home/Home.kt#L230
</details>

### PR DESCRIPTION
**Pull request by MV-GH**
According to the m3 specs, dropdown has a 3dp tonal elevation, this was missing and caused the discrepancy between DropdownMenu and CascadeDropdownMenu. See https://m3.material.io/styles/elevation/tokens#df721a00-888e-4c5e-bfe1-5d905f167aaa

After:

https://github.com/saket/cascade/assets/67873169/4d59acd9-4159-4c2a-834e-d3bdc0d9d5e8

Before:

https://github.com/saket/cascade/assets/67873169/4008fd78-c6d5-42e7-9202-a39f8f033dcd

Fixes #39

### CODE REVIEW

**Review by saket — APPROVED** [review]
Thank you so much!

**Comment by saket** [comment]
Can you regenerate test screenshots using and add them to this PR?

```bash
./gradlew cascade-compose:connectedDebugAndroidTest -Pdropshots.record
```

**Comment by MV-GH** [comment]
I generated them using a Pixel 7 API 31.

**Comment by MV-GH** [comment]
Made some small changes, I saw that it was setting the shadowElevation to the tonalElevation. And made a argument for tonalElevation

**Comment by MV-GH** [comment]
There was a test with this remark

>   // Drop shadows are strangely causing tiny differences in
  // generated screenshots. A zero elevation is used to avoid this.
  private val shadowElevation = 0.dp

This was actually changing the tonalElevation and now it actually changes the shadowElevation

Not sure if this is still needed?

**Comment by saket** [comment]
I'm sorry this took me some time. `trunk` has moved ahead so let me land your changes manually.

File: cascade-compose/src/main/java/me/saket/cascade/Cascade.kt [hunk 1]
```diff
@@ -118,6 +119,7 @@ fun CascadeDropdownMenu(
   offset: DpOffset = DpOffset.Zero,
   fixedWidth: Dp = 196.dp,
   shadowElevation: Dp = 3.dp,
+  tonalElevation: Dp = 3.dp,
```
Comment (saket) [inline cascade-compose/src/main/java/me/saket/cascade/Cascade.kt:122]: Do you need your popups to have a different shadow and tonal elevations?

Comment (MV-GH) [reply cascade-compose/src/main/java/me/saket/cascade/Cascade.kt:122]: I don't, but others might and this matches the Compose API as it gives both options

Comment (saket) [reply cascade-compose/src/main/java/me/saket/cascade/Cascade.kt:122]: Can you point me to that? I don't see any `tonalElevation` parameter on `DropdownMenu()`:

https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/material/material/src/androidMain/kotlin/androidx/compose/material/AndroidMenu.android.kt;l=112;drc=cdd989b146a4758f9974b63c2e53e4eb767e961e

Comment (saket) [reply cascade-compose/src/main/java/me/saket/cascade/Cascade.kt:122]: Sorry wrong link, here's m3:

https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/material3/material3/src/androidMain/kotlin/androidx/compose/material3/AndroidMenu.android.kt;l=84;drc=15aff086d739e6673f71e5cc7bf0d01689942cc6

**Pull request closed without merging**